BlueprintUICommonControls/Sources/AttributedLabel.swift

```diff
@@ -31,7 +31,10 @@ public struct AttributedLabel: Element {
             }
         }
 
-        return ElementContent(measurable: Measurer(attributedText: attributedText, roundingScale: roundingScale))
+        return ElementContent(
+            measurable: Measurer(attributedText: attributedText, roundingScale: roundingScale),
+            measurementCachingKey: .init(type: AttributedLabel.self, input: self)
```

Comment:
I think you can use `Self` here now, which just seems "nicer" IMO (and doesn't need updating if the name of the type changes)

```suggestion
            measurementCachingKey: .init(type: Self, input: self)
```

Reply:
Ah right, I always forget where this works; since it gradually got more and more permeated through a few versions of Swift.